libs/java/server_common/src/main/java/com/yahoo/athenz/common/ServerCommonConsts.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 Verizon Media
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.yahoo.athenz.common;
+
+public final class ServerCommonConsts {
+    public static final String ADMIN_ROLE_NAME  = "admin";
```

Comment:
Constants that also exist in ZMS project. Prefered to avoid deleting them and redirecting to the common constants at this point as it will require changes in a lot of files (redirecting them to the common version).